staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/skeleton.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schema
```

Comment:
Could we move `Unflod` method (`func (s *Structural) Unfold() *Structural {}`) to this file and then remove `unfold.go` file?

Reply:
no, it's not about removing anything